src/Microsoft.Framework.CommandLineUtils.Sources/project.json

```diff
@@ -1,12 +1,14 @@
 {
     "version": "1.0.0-*",
-    "compilationOptions" : { "define" : ["TRACE"], "warningsAsErrors": true },
+    "compilationOptions": {
+        "define": [ "TRACE" ],
+        "warningsAsErrors": true
+    },
     "shared": "**/*.cs",
-    "frameworks" : {
-        "dnx451" : { },
-        "dnxcore50" : { 
+    "frameworks": {
+        "dnx451": { },
+        "dnxcore50": {
             "dependencies": {
-                "System.Collections": "4.0.11-beta-*",
                 "System.Runtime": "4.0.21-beta-*"
```

Comment:
I get a build error saying `error CS8021: No value for RuntimeMetadataVersion found. No assembly containing System.Object was found nor was a value for RuntimeMetadataVersion specified through options.`